docs/source/configuration/multilingual.md

```diff
@@ -28,22 +29,37 @@ config.settings = {
 }
 ```
 
-Declare the language you want to support in your site in the `supportedLanguages` array,
-and which is the default language of your site.
+Declare the language you want to support in your site in the `supportedLanguages` array, and which is the default language of your site.
 
 ```{warning}
-The default language and the supported languages must match the one set in the Plone
-side, and those should be set using GenericSetup using your policy package, or
-manually via the Languages control panel, i.e. en for English, or pt-br for Portuguese (Brazil)
+The default language and the supported languages must match those set in the Plone backend.
+Those should be set either using GenericSetup using your policy package or manually via the {guilabel}`Languages` control panel.
+Examples include `en` for English, or `pt-br` for Portuguese (Brazil).
+```
+
+## Docker backend configuration
```

Comment:
@stevepiercy I think this is (arguably) the wrong place for this documentation. This chapter is about multilingual features, which means supporting more than one content language in the same Plone site. The LANGUAGE environment variable determines the first (and only) language for a new Plone site created by the plone-backend image, and it does not enable the multilingual features.

I think https://6.docs.plone.org/volto/contributing/developing-core.html#start-the-backend-and-volto is the most important place to mention the LANGUAGE option narrative docs.

Reply:
Agreed, thanks for finding a better home for it. I moved it there, and rearranged some bits and pieces to make it flow better and be easier to navigate.